src/ElectricPokemon.java

```diff
@@ -0,0 +1,139 @@
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public class ElectricPokemon extends Pokemon {
+    private final String name;
```

Comment:
Deze variabelen had je beter in de Pokemon class kunnen definieren. Nu heb je ze in elke subclass opnieuw moeten definieren, terwijl het ook 1x in de super kan